sonar-application/src/main/java/org/sonar/application/ClusterProperties.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonar.application;
+
+import org.apache.commons.lang3.StringUtils;
+import org.sonar.process.Props;
+
+/**
+ * Properties of the cluster configuration
+ */
+final class ClusterProperties {
+
+  class Parameters {
+    static final String ENABLED = "sonar.cluster.enabled";
+    static final String MEMBERS = "sonar.cluster.members";
+    static final String PORT = "sonar.cluster.port";
+    static final String PORT_AUTOINCREMENT = "sonar.cluster.port_autoincrement";
+    static final String INTERFACES = "sonar.cluster.interfaces";
+    static final String NAME = "sonar.cluster.name";
+    static final String LOG_LEVEL = "sonar.cluster.log.level";
+  }
+
+  private int port;
+  private boolean enabled;
+
+  private boolean portAutoincrement;
+  private String members;
+  private String interfaces;
+  private String name;
+  private String logLevel;
+
+  ClusterProperties(Props props) {
+    port = props.valueAsInt(Parameters.PORT, getDefaultPort());
+    enabled = props.valueAsBoolean(Parameters.ENABLED, false);
+    portAutoincrement = props.valueAsBoolean(Parameters.PORT_AUTOINCREMENT, false);
+    interfaces = props.value(Parameters.INTERFACES, "");
+    name = props.value(Parameters.NAME, "");
+    logLevel = props.value(Parameters.LOG_LEVEL, "WARN");
+    members = props.value(Parameters.MEMBERS, "");
+  }
+
+  int getPort() {
+    return port;
+  }
+
+  boolean isEnabled() {
+    return enabled;
+  }
+
+  boolean isPortAutoincrement() {
+    return portAutoincrement;
+  }
+
+  String getMembers() {
+    return members;
+  }
+
+  String getInterfaces() {
+    return interfaces;
+  }
+
+  String getName() {
+    return name;
+  }
+
+  String getLogLevel() {
+    return logLevel;
+  }
+
+  int getDefaultPort() {
+    return 9003;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this method and declare a constant for this value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3400)